scripts/validate-merge-group-heal/main.go

```
@@ -0,0 +1,136 @@
+			return "", false
+		}
+		return strings.Join(conditionLines, "\n"), true
+	}
+
+	return "", false
+}
+
+func run(workflowPath string, stdout io.Writer, stderr io.Writer) int {
+	workflow, err := os.ReadFile(workflowPath)
+	if err != nil {
+		fmt.Fprintf(stderr, "merge-group heal contract: read workflow: %v\n", err)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)